server/rails.gradle

```diff
@@ -135,6 +135,16 @@ task compileAssets(type: ExecuteUnderRailsTask) {
   }
 
   doLast {
+	project.exec {
```

Comment:
I believe this should not be needed since there's already a dependency on `npmInstall` around line 113.

`npmInstall` should ideally be called `yarnInstall` since we've moved over to yarn, but that's a separate issue.

If you can remove this block of code, and squash the PR. I'm happy to merge it.